documentation/modules/exploit/windows/http/ektron_xslt_exec_ws.md

```diff
@@ -0,0 +1,70 @@
+## Vulnerable Application
+
+  Ektron Content Management System (CMS) 8.5 and 8.7 before 8.7sp2 and 9.0 before sp1 (according to CVE-2015-0923)
+
+## Verification Steps
+
+  Ektron has been acquired by another company and finding installers may prove difficult. But if you can build a test network the following may be used to verify:
+
+  1. Install the application
+  2. Start msfconsole
+  3. Do: ```use exploit/windows/http/ektron_xslt_exec_ws```
+  4. Do: ```set RHOST [target ip]```
+  5. Do: ```check```
+  6. You should receive a target vulnerable message.
+  7. Do: ```run```
```

Comment:
`run` will work, but may want to change to `exploit` since this is an exploit and not an aux module